### DISCUSSION TITLE
Translation of the Card

### DISCUSSION THREAD

**Discussion by mekki1986** [vasqued2/ha-teamtracker-card#79] [General]
Hi, first of all thank you for the great integration and sorry for my English.
I can't translate the card well. I set up the API and everything works correctly, but I have part of the card in Italian (for example the time until the match) and some in English (Today, kickoff etc.) I followed the instructions in the readme.rd file in the folder 'locate'. I created the it.js file with the translation. Then in the translator.js file I imported the language and inserted it in the list below where there is "Var languages".
But the card always remains the same.
What step am I missing to solve my problem?
Thank you all

**Comment by vasqued2** [comment]
If you added both the import and var lines, try clearing the cache and refreshing.
